docs/deploy.md

```diff
@@ -120,3 +120,9 @@ The deploy plugin will now stage this file to the `bin` directory with run-class
     $ ./servicename.sh restart # Same as 'stop' followed by 'start'.
 
 This will also use `servicename` for your logback appname.
+
+If you need to pass arguments to the service's main class on startup, you can do so by appending
+the args to the call to `servicename.sh`, separated from the `start|restart` command by `--`:
+
+    $ ./servicename.sh start -- arg1 arg2 --flag
```

Comment:
Rethinking this a little: I don't think it's a good idea to require flags to be passed to these. If we require things like `--port` to be specified on the commandline, it means that we can't start or stop services in production without looking into our deployment configuration.

We should look into either using environment variables or system properties to pass in the port (which typesafe config supports), or look into auto-generating these start / stop scripts from a template.

:/